reference/nodejs.md

```diff
@@ -0,0 +1,90 @@
+# Node.js
+
+Node.js is a popular JavaScript runtime built on Chrome's V8 JavaScript engine.
+ Platform.sh supports deploying Node.js applications quickly and easily.
+
+## Supported versions
+
+* 0.12.7 (default)
+
+## Configuration
+
+To use Platform.sh and Node.js together, configure the ``.platform.app.yaml``
+file with a few key settings, as described here (a complete example is included
+at the end).
+
+1. Specify the `type` as ``nodejs``
+2. Specify your dependencies under the `nodejs` key, like this:
+
+   ```yaml
+   dependencies:
+     nodejs:
+       'pm2': "*"
+       'react': "^0.13.0"
+       'url-loader': "~0.5.5"
+   ```
+   If you have a `package.json` file at the root of your project you can
```

Comment:
No, `npm install` should be run in the build hook, which will install the dependencies specified in the `package.json`